README.md

```diff
@@ -6,6 +6,7 @@ The Bookstore is a website that allows the user to :
 -Add a book.
```

Comment:
- [ ] [Optional] Kindly fix your packages, as for now you can't install the node packages without the --force prefix `npm Install --force`. you can check your react to see if its old version is the cause of these errors